### PR TITLE
add long description for verify-install

### PR DESCRIPTION
**Pull request by irisdingbj**

### CODE REVIEW

**Comment by esnible** [comment]
I have general feedback on the command itself.

The output "istio is installed successfully" should have Istio capitalized.

The command's help reports `--filename stringSlice` and doesn't have a description of what --filename is.  It would be better if it reported something like `--filename string    Istio YAML configuration file`.

I am not sure what the command does.  I built from source using release-1.1 branch and did `~/go/out/darwin_amd64/release/istioctl experimental verify-install -f ~/istio.yaml` using an istio.yaml that was built from `helm template install/kubernetes/helm/istio --name istio --namespace istio-system --set global.mtls.enabled=true --set global.proxy.enableCoreDump=true > $HOME/istio.yaml`.  It did some thinking and 15 seconds later reported success.  However I did the same thing passing in a file generated from `helm template install/kubernetes/helm/istio-init --name istio --namespace istio-system > $HOME/istio-init.yaml` which reported success in 1 second.  Yet this second file is very different from the first.  Perhaps this command should output what it has checked?

**Comment by irisdingbj** [comment]
@esnible  Thanks for the review.  

1. change all istio->Istio as you suggested.
2. Add description for file name in help
3. Fix grammar error.

As the resources in Istio installation file might be huge, so if we report what has checked one by one it will be flush the screen.  It will report the specific resource if it is not ready.

**Comment by esnible** [comment]
> As the resources in Istio installation file might be huge, so if we report what has checked one by one it will be flush the screen. It will report the specific resource if it is not ready.

Consider adding messages `Deployment %s checked.` and `Job %s checked`.  I did that and got the output:

```
Job istio-cleanup-secrets-release-1.1-20190214-09-16 checked.
Deployment istio-galley checked.
Deployment istio-egressgateway checked.
Deployment istio-ingressgateway checked.
Deployment istio-ingress checked.
Deployment istio-policy checked.
Deployment istio-telemetry checked.
Deployment istio-pilot checked.
Deployment prometheus checked.
Deployment istio-citadel checked.
Deployment istio-sidecar-injector checked.
istio is installed successfully
```

Seeing which deployments were being checked gave me more confidence that I used the command properly.  I tried passing in `-f ~/istio.yaml` and `-f ~/istio-init.yaml` and got the expected results.

I also tried `-f ~/istio.yaml ~/istio-init.yaml` and `-f ~/istio.yaml -f ~/istio-init.yaml`.  The first one did not check the jobs of _istio-init.yaml_ and it didn't report a syntax problem either.  The user might appreciate it if the tool complains about syntax if there are positional (non-flagged) arguments.

I tried `-f samples/bookinfo/platform/kube/bookinfo.yaml`.  It reported that Istio was installed correctly.  Because it showed the pods at least I had some idea something was wrong.  It might be worthwhile to count how many deployments beginning with "istio-" are found in the .yaml and report something like "0 Istio deployments found".

File: istioctl/pkg/install/verify.go [hunk 1]
```diff
@@ -138,13 +138,17 @@ func NewVerifyCommand() *cobra.Command {
...
 			Recursive: boolPtr(true),
+			Usage:     "Istio YAML installation file.",
 		}
 	)
 	verifyInstallCmd := &cobra.Command{
 		Use:   "verify-install",
 		Short: "Verifies Istio Installation Status",
 		Long: `
-		verify-install Verifies Istio Installation Status
+		verify-install Verifies Istio Installation Status against the installation file
```
Comment (linsun) [inline istioctl/pkg/install/verify.go]: change to verifies Istio installation status

File: istioctl/pkg/install/verify.go [hunk 2]
```diff
@@ -138,13 +138,17 @@ func NewVerifyCommand() *cobra.Command {
...
+		you specified when you installed Istio. It loops through all the installation
+		resources defined in your installation files and reports whether all of them are
```
Comment (linsun) [inline istioctl/pkg/install/verify.go]: is it one installation file or multiple?  you had singular earlier.

File: istioctl/pkg/install/verify.go [hunk 3]
```diff
@@ -138,13 +138,17 @@ func NewVerifyCommand() *cobra.Command {
...
+		in ready status. It will report failure when any of them are not ready.
```
Comment (linsun) [inline istioctl/pkg/install/verify.go]: could you also submit a PR on using this cmd at the end of the install doc?   

https://istio.io/docs/setup/kubernetes/helm-install/
https://istio.io/docs/setup/kubernetes/quick-start/

Comment (linsun) [reply istioctl/pkg/install/verify.go]: also does this cmd work with the new CRDs which are in istio-init now?

File: istioctl/pkg/install/verify.go (hunk 1 above)
Comment (irisdingbj) [reply istioctl/pkg/install/verify.go]: done

File: istioctl/pkg/install/verify.go (hunk 2 above)
Comment (irisdingbj) [reply istioctl/pkg/install/verify.go]: singular and change to all to singular

File: istioctl/pkg/install/verify.go (hunk 3 above)
Comment (irisdingbj) [reply istioctl/pkg/install/verify.go]: yes

Comment (linsun) [reply istioctl/pkg/install/verify.go]: cool!

**Comment by linsun** [comment]
@wenchenglu can u please merge this in 1.1?

**Pull request merged**